website/templates/project/registrations.mako

```diff
@@ -22,23 +35,173 @@
     ##      %else:
     ##          There have been no registrations of the parent project (<a href="${parent_node['url']}">${parent_node['title']}</a>).
     ##      %endif
-    % else:
-        <div class="m-md">
-            There have been no registrations of this ${node['node_type']}.
-            For a list of the most viewed and most recent public registrations on the
-            Open Science Framework, click <a href="/explore/activity/#newPublicRegistrations">here</a>.
-        </div>
-    % endif
-    %if parent_node['exists'] and user['is_admin_parent']:
-        <div class="m-l-md">To register the entire project "${parent_node['title']}" instead, click <a href="${parent_node['registrations_url']}">here.</a></div>
-    %endif 
+        % else:
+        <p>
+          There have been no completed registrations of this ${node['node_type']}.
+          For a list of the most viewed and most recent public registrations on the
+          Open Science Framework, click <a href="/explore/activity/#newPublicRegistrations">here</a>,
+          or you start a new draft registration from the "Draft Registrations" tab.
+        </p>
+        % endif
+        %if parent_node['exists'] and user['is_admin_parent']:
+        <br />
+        <br />
+        To register the entire project "${parent_node['title']}" instead, click <a href="${parent_node['registrations_url']}">here.</a>
+        %endif
+      </div>
 
-  </div>
-  <div class="col-sm-3">
-    <div class="m-md">
         % if 'admin' in user['permissions'] and not disk_saving_mode:
-          <a id="registerNode" href="${node['url']}register" class="btn btn-success" type="button">New Registration</a>
+            <div class="col-md-3">
+        <a data-bind="click: createDraftModal, css: {disabled: loading}" id="registerNode" class="btn btn-default" type="button">
+          New Registration
+        </a>
+            </div>
         % endif
     </div>
   </div>
+  <div role="tabpanel" class="tab-pane" id="drafts">
+    <div class="row" style="min-height: 150px;padding-top:20px;">
+      <div data-bind="visible: !preview()">
+        <div class="col-md-9">
+          <div class="scripted" data-bind="foreach: drafts">
+            <li class="project list-group-item list-group-item-node">
+              <h4 data-bind="text: schema().title" ></h4>
+              <h4 class="list-group-item-heading">
+                <div class="progress progress-bar-md">
+                  <div class="progress-bar" role="progressbar" aria-valuemin="0" aria-valuemax="100"
+                       data-bind="attr.aria-completion: completion,
+                                  style: {width: completion() + '%'}">
```

Comment:
In some initial functional review, I could not find a progress bar on the page. Is this latent functionality (intended for prereg, but not currently in use)?